internal/queue/queue.go

```diff
@@ -48,12 +50,23 @@ func (q *RabbitQueue) Status() map[string]map[string]bool {
 	return status
 }
 
-func (q *RabbitQueue) Run(serviceManager *service.ServiceManager) {
-	q.processActors(serviceManager)
-	q.processObjects(serviceManager)
+func (q *RabbitQueue) Run(ctx context.Context, serviceManager *service.ServiceManager, wg *sync.WaitGroup) {
+	wg.Add(2)
+
+	go func() {
+		defer wg.Done()
+		q.logger.Info("starting actor consumer")
+		q.processActors(ctx, serviceManager)
+	}()
+
+	go func() {
+		defer wg.Done()
+		q.logger.Info("starting object consumer")
```

Comment:
We should either make these both debug, or call this something other than "object" (which I kind of think we should rename in the code anyway, I just don't have a better name right now)